.github/workflows/test_pull_requests.yml

```diff
@@ -70,7 +70,18 @@ jobs:
           pip install --upgrade pip semgrep
           # f"..." and f'...' are the same for semgrep
           semgrep --error --lang python --pattern 'trans._(f"...")' napari
-          semgrep --error --lang python --pattern 'trans._($X.format(...))' napari
+          semgrep --error --lang python --pattern "trans._(\$X.format(...))" napari
+
+  actionlint:
+    name: Action lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Check workflow files
+        run: |
+          bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash)
+          ./actionlint -color -ignore SC2129
+        shell: bash
```

Comment:
This is a bit icky (downloading a bash script from main). Maybe put it in its own workflow? Ideally it would be its own action. Also, it should only run when the .github folder is touched?

Reply:
This is a official way to do this based on documentation. The script itself is downloading latest release. 

Moved to won action,